src/rulesets/oas3/index.json

```
@@ -52,6 +52,19 @@
         "operation"
       ]
     },
+    "oas3-parameter-description": {
+      "description": "Parameter objects should have a `description`.",
+      "recommended": true,
+      "type": "style",
+      "given": "$..[?(@parentProperty !== 'links' && @.parameters)]['parameters']..[?(@.in)]",
```

```suggestion
      "given": "$..[?(@parentProperty !== 'links' && @.parameters)]['parameters'].[?(@.in)]",
```

Maybe check if single `dot` is enough - I think it should be.